prac_06/guitars.py

```diff
@@ -0,0 +1,15 @@
+from prac_06.guitar import Guitar
```

Comment:
Again, I put a docstring in this file marking it as the client code - although it gets a bit tricky as to where we record our time! (I made a hash of that myself ;-))